libs/application/ui-forms/src/lib/applicantInformationSection/messages.ts

```diff
@@ -0,0 +1,48 @@
+import { defineMessages } from 'react-intl'
+
+export const applicantInformation = {
+  general: defineMessages({
+    title: {
+      id: 'ais.application:applicantInfo.general.title',
+      defaultMessage: 'Upplýsingar um þig',
+      description: 'Information about you',
+    },
+  }),
+  labels: defineMessages({
+    name: {
+      id: 'ais.application:applicantInfo.labels.name',
```

Comment:
Needs a different namespace. Either one for all shared ui-forms or one per each.